src/org/benf/cfr/reader/bytecode/analysis/structured/statement/StructuredTry.java

```diff
@@ -77,8 +78,15 @@ public Dumper dump(Dumper dumper) {
             dumper.separator(")");
         }
         tryBlock.dump(dumper);
-        for (Op04StructuredStatement catchBlock : catchBlocks) {
+        dumper.removePendingCarriageReturn();
```

Comment:
This can be achieved just by having dumper.removePendingCarriageReturn().separator(" ") 

Here